playbooks/common/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,30 @@
+---
+# NOTE: ideally this would be just part of a single play that lists the existing
+# checks when openshift_checks is not set or run the requested checks. However,
+# to actually run the checks we need to have the included dependencies to run
+# first and that takes time. To speed up listing checks, we use a separate play
+# that runs before the include to save time and improve the UX.
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
I think yes, run it against localhost so we don't have to init groups at this point nor put `OSEv3` in a `common` playbook. The only inconsistency would be if they didn't define `openshift_checks` globally but only on individual hosts, and I think that's not a very important use case.